packages/gatsby-remark-katex/package.json

```diff
@@ -29,7 +32,7 @@
   "main": "index.js",
   "peerDependencies": {
     "gatsby": "^3.0.0-next.0",
-    "katex": "^0.12.0"
+    "katex": "^0.13.3"
```

Comment:
current version is `^0.13.11`, but I'm not sure if it is convention to keep it at the lowest patch version (i.e. `^0.13.0`). Maybe change it to  `^0.13.11`? Or just leave it as is.